setup.py

```diff
@@ -21,7 +21,7 @@
     "scikit-learn>=0.23.2",  # sure
     "scipy>=1.5.4",  # in factorization API only
     "tensorly>=0.5.1",  # in factorization and model_weights API only
-    "torch==2.0.0",  # also change the version in the test.yaml when changing this next time
+    "torch==2.3.0",  # also change the version in the test.yaml when changing this next time
```

Comment:
Please add a reminder here to update the pytorch version in the bindingdb_deepdta tutorial notebook.